### PR TITLE
[ODS-4814] Bring in APIModel changes for API version

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

**Comment by semalaiappan** [comment]
https://intedfitools1.msdf.org/buildConfiguration/OdsPlatform_OdsApi_Packages_EdFiOdsApiTestSdk?branch=ODS-4814&mode=builds#all-projects EdFi.Ods.Api.TestSdk build is succeed in this branch

File: Samples/Extensions/StudentTranscript/StudentTranscriptMetaEd/MetaEdOutput/SampleStudentTranscript/ApiMetadata/ApiModel-EXTENSION.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "odsApiVersion": "5.1.0",
+  "odsApiVersion": "5.2.0",
```
Comment (tbmeador-dlp) [inline Samples/Extensions/StudentTranscript/StudentTranscriptMetaEd/MetaEdOutput/SampleStudentTranscript/ApiMetadata/ApiModel-EXTENSION.json:2]: we need to use the latest dev version of MetaEd 2.4.2-dev.3 targeting version v5.2 which has the changes from [METAED-1186](https://tracker.ed-fi.org/browse/METAED-1186). This should change all the `odsApiVersion` properties into `5.2`.

**Review by tbmeador-dlp — CHANGES_REQUESTED** [review]

**Comment by tbmeador-dlp** [comment]
> [intedfitools1.msdf.org/buildConfiguration/OdsPlatform_OdsApi_Packages_EdFiOdsApiTestSdk?branch=ODS-4814&mode=builds#all-projects](https://intedfitools1.msdf.org/buildConfiguration/OdsPlatform_OdsApi_Packages_EdFiOdsApiTestSdk?branch=ODS-4814&mode=builds#all-projects) EdFi.Ods.Api.TestSdk build is succeed in this branch

Need to make sure that staging is loading the extensions. An easy way to do this is to check the [api version endpoint](https://api-stage.ed-fi.org/v5.2/api) and look at the `dataModels`. The build may pass either way but if staging doesn't have the extensions neither will the generated sdk.

File: Samples/Extensions/StudentTranscript/StudentTranscriptMetaEd/MetaEdOutput/SampleStudentTranscript/ApiMetadata/ApiModel-EXTENSION.json (hunk 1 above)
Comment (semalaiappan) [reply Samples/Extensions/StudentTranscript/StudentTranscriptMetaEd/MetaEdOutput/SampleStudentTranscript/ApiMetadata/ApiModel-EXTENSION.json:2]: I took latest from MetaEd and updated now 
https://intedfitools1.msdf.org/buildConfiguration/OdsPlatform_MetaEd_DataStandard_MetaEdJsDataStandardStaging33/122694?buildTab=artifacts

**Comment by semalaiappan** [comment]
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Extensions  PR  is top of  4809 .  I will update to main repo after merge of 4809

**Comment by semalaiappan** [comment]
> > [intedfitools1.msdf.org/buildConfiguration/OdsPlatform_OdsApi_Packages_EdFiOdsApiTestSdk?branch=ODS-4814&mode=builds#all-projects](https://intedfitools1.msdf.org/buildConfiguration/OdsPlatform_OdsApi_Packages_EdFiOdsApiTestSdk?branch=ODS-4814&mode=builds#all-projects) EdFi.Ods.Api.TestSdk build is succeed in this branch
> 
> Need to make sure that staging is loading the extensions. An easy way to do this is to check the [api version endpoint](https://api-stage.ed-fi.org/v5.2/api) and look at the `dataModels`. The build may pass either way but if staging doesn't have the extensions neither will the generated sdk.

image

**Pull request merged**